packages/@sanity/vision/src/components/NoResultsDialog.js

```diff
@@ -1,17 +1,28 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import {Box, Heading, Text, Stack} from '@sanity/ui'
+import styled from 'styled-components'
 
+const QueryContainer = styled.pre`
+  margin: 0;
+`
 function NoResultsDialog(props) {
   return (
-    <div className="vision_no-results">
-      <h1>No matches</h1>
-      <p>
-        No documents found in dataset <code>{props.dataset}</code> that match query:
-      </p>
-      <pre>
-        <code>{props.query}</code>
-      </pre>
-    </div>
+    <Box paddingY={2} className="vision_no-results">
+      <Heading as="h1" size={5}>
```

Comment:
Would prefer no custom size on this header:
```suggestion
      <Heading as="h1">
```